packages/digit_dss/lib/widgets/dashboard/dashboard_metric_card.dart

```diff
@@ -0,0 +1,167 @@
+                            style: theme.textTheme.bodyMedium
+                                ?.apply(color: theme.colorScheme.secondary),
+                          ),
+                          const Padding(
+                            padding: EdgeInsets.only(right: 3.0),
+                            child: Visibility(
+                              child: Icon(Icons.arrow_drop_down),
+                            ),
+                          )
+                        ]),
+                  ),
+                ),
+              )
+            ],
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: kPadding, bottom: kPadding),
+            child: Wrap(
+              spacing: 2.0, // Space between items
+              runSpacing: 2.0, // Space between lines
+              alignment: WrapAlignment.start,
+              children: dashboardState.maybeWhen(
+                  orElse: () => [],
+                  fetched: (metricData, tableData, selectedDate) {
+                    return metricData != null
+                        ? metricData.entries.mapIndexed((
+                            i,
+                            entry,
+                          ) {
+                            return SizedBox(
+                              height: MediaQuery.of(context).size.width / 3.8,
+                              child: Row(
+                                mainAxisAlignment: MainAxisAlignment.center,
+                                mainAxisSize: MainAxisSize.min,
+                                children: [
+                                  buildMetric(
+                                    context,
+                                    entry.key,
+                                    entry.value.value,
+                                    i,
+                                    localizations,
+                                  ),
+                                  if (entry.key != metricData.entries.last.key)
+                                    VerticalDivider(
+                                      width: kPadding,
+                                      color: DigitTheme.instance.mobileTheme
+                                          .colorScheme.outline,
+                                      thickness: 2,
+                                    ),
+                                ],
+                              ),
+                            );
+                          }).toList()
+                        : [];
+                  }),
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Padding(
+              padding: const EdgeInsets.symmetric(vertical: kPadding),
+              child: Text(
+                '5 Spray operators synced atleast once',
```

Comment:
Why is this static? 

And add localization

Reply:
This chart is yet to be finalized